util/slices.go

```diff
@@ -0,0 +1,13 @@
+package util
+
+// IndexOfString is a general utility function that can find the index of a value
+// present in a string slice. The second value is true if the item is found.
+func IndexOfString(slice []string, value string) (int, bool) {
```

Comment:
This is probably due to general newness to go and it's structures and syntax but it took me a while to realize that this was testing for the presence of an element in a collection of elements rather than the presence of a substring within a string. Probably baggage from character arrays and the concept of a slice of a string from Perl combined with previous indexOf type functions. I don't think it has to change I just thought I'd mention it.

This is relevant because commands of the form of `util.Command("bash", "-c", fmt.Sprintf("echo 'nameserver %s' | sudo tee /etc/resolver/vm", bridgeIP)).Run()` on 178 of dns.go would fail this check. It's not really an issue for Macs because that one is guaranteed to be proceeded by a command which will already trigger the Password prompt and stop the spinner (line 175 of dns.go). I think it might affect Linux however as the configureLinuxResolver function has no such preceding command.

I've attached an example of what this looks like on Linux.

![privilege-prompt-test-results-linux](https://user-images.githubusercontent.com/223075/35624191-3f10e0a6-065c-11e8-94c2-5bf94fa62432.png)

Reply:
One potential avenue around this would be to import the strings package and change the conditional on line 7 to something like
`		if elem == value || strings.Contains(elem, value + " ") {`

This appears to function properly for both mac and linux though it changes the nature of this function such that you'll probably want to rename it.

Reply:
There's a convention with exec.Cmd that the tokens all be individual array elements. I see that subshell situations might be different. Will adjust.